src/Umbraco.Web.UI/Umbraco/config/lang/en_us.xml

```diff
@@ -2353,6 +2353,7 @@ To manage your website, simply open the Umbraco backoffice and start adding cont
       <key alias="labelUsedInMediaDescendants">One or more of this item's descendants is being used in a media item</key>
       <key alias="labelUsedInContentDescendants">One or more of this item's descendants is being used in a content item</key>
       <key alias="labelUsedInMemberDescendants">One or more of this item's descendants is being used in a member</key>
+      <key alias="deleteWarning">The current item or one or more of this item's descendants is being used. Deleting this item can lead to broken links on your website.</key>

```

Comment:
I think it would be better with a little shorter text, as its easier to read, something like this:

"This or its descendants is being used. Deleting can lead to broken links on your website."